corehq/apps/sms/tests/__init__.py

```diff
@@ -392,4 +395,14 @@ def test_backend(self):
         self.backend3.delete_invoke_doc()
         self.assertFalse(self.backend3.invoke_doc_exists())
 
+    def test_sms_registration(self):
+        incoming("+9991234567", "JOIN {} WORKER tester".format(self.domain), "BACKEND1")
```

Comment:
@TylerSheffels I misspoke, it should actually be "TEST_CASE_BACKEND".  The third arg to incoming (https://github.com/dimagi/commcare-hq/blob/register-mobile-worker-sms/corehq/apps/sms/api.py#L284) is the backend_api, which is just a string that identifies the type of backend.  So while "BACKEND1" is the name of the specific backend, we really want to pass in the api_id defined here: https://github.com/dimagi/commcare-hq/blob/register-mobile-worker-sms/corehq/apps/sms/tests/__init__.py#L22